moz_minions/kevin.py

```diff
@@ -177,6 +179,31 @@ def check_process_exist(self):
                 return True
         return False
 
+    def remove_exist_lock_file(self, serial_no):
+        lock_file_path = os.path.join("/tmp/LOCKS", serial_no + ".lock")
+        if os.path.exists(lock_file_path):
+            os.remove(lock_file_path)
+
+    def exec_process(self, cmd_list):
+        proc = subprocess.Popen(cmd_list, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        output = proc.communicate()[0]
+        if proc.stdin:
+            proc.stdin.close()
+        if proc.stdout:
+            proc.stdout.close()
+        if proc.stderr:
+            proc.stderr.close()
+        return output
+
+    def remove_adb_forward_list(self, serial_no):
```

Comment:
I've considered this option, but it seems we don't have forward argument support in adb helper...but I still can go that way ....but that will take a little bit longer to deliver these code~
how do you think?